viso_odom_calculator:
  rate: 10
  force_feedback_sigma: 1.0
  distribution_feedback_minimum_sigma: 0.1
  max_feedback_time: 0

biped_particle_odometry:
  particle_num: 50
  rate: 100
  use_imu: false

viso_odometry_offset:
  use_twist_filter: false
  overwrite_pdf: true
  twist_proportional_sigma: false
  sigma_x: 0.05
  sigma_y: 0.05
  sigma_z: 0.3
  sigma_roll: 0.3
  sigma_pitch: 0.3
  sigma_yaw: 0.05
  mean_x: 0.0
  mean_y: 0.0
  mean_yaw: 0.0

biped_odometry_offset:
  use_twist_filter: true
  filter_buffer_size: 10
  calculate_covariance: true
  twist_proportional_sigma: false
  overwrite_pdf: true
  sigma_x: 0.1
  sigma_y: 0.1
  sigma_z: 0.001
  sigma_roll: 0.001
  sigma_pitch: 0.001
  sigma_yaw: 0.1
  mean_x: 0.01
  mean_y: 0.0
  mean_yaw: 0.0